homeassistant/components/volvooncall/config_flow.py

```diff
@@ -0,0 +1,90 @@
+"""Config Flow Handler for volvooncall."""
+import voluptuous as vol
+from volvooncall import Connection
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_REGION, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+from . import VolvoData
+from .const import CONF_MUTABLE, CONF_SCANDINAVIAN_MILES, DOMAIN
+from .errors import AuthenticationError, InvalidRegionError
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_REGION): cv.string,
+        vol.Optional(CONF_MUTABLE, default=True): cv.boolean,
+        vol.Optional(CONF_SCANDINAVIAN_MILES, default=False): cv.boolean,
+    },
+)
+
+
+class VolvoOnCallConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """VolvoOnCall config flow."""
+
+    VERSION = 1
+    data = None
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
```

Comment:
Please type the whole signature when adding type annotations. Did you run our scaffold script to generate the config flow template? It will add type annotations by default.